docs/podman-container.1.md

```diff
@@ -14,8 +14,8 @@ The container command allows you to manage containers
 | Command    | Man Page                                            | Description                                                                  |
 | ---------  | --------------------------------------------------- | ---------------------------------------------------------------------------- |
 | attach     | [podman-attach(1)](podman-attach.1.md)              | Attach to a running container.                                               |
-| checkpoint | [podman-container-checkpoint(1)](podman-container-checkpoint.1.md)  | Checkpoints one or more containers.                          |
-| cleanup    | [podman-container-cleanup(1)](podman-container-cleanup.1.md)    | Cleanup containers network and mountpoints.                      |
+| checkpoint | [podman-container-checkpoint(1)](podman-container-checkpoint.1.md)  | Checkpoints one or more running containers.                  |
+| cleanup    | [podman-container-cleanup(1)](podman-container-cleanup.1.md)    | Cleanup container network and mountpoints.                       |
```

Comment:
suggest:
s/container network/the container's network/
Also you've an extra space at the end of the line that will be problematic for the CI checks.

Reply:
Oops, no extra space, you've an ending bar that just <cr>'d on me in the github review.